assets/js/components/DatabasesOverview/DatabaseItemOverview.jsx

```diff
@@ -15,22 +15,18 @@ const databaseInstanceColumns = [
   { name: 'System Replication' },
   { name: 'Cluster' },
   { name: 'Host' },
+  { name: ' ' },
```

Comment:
Do we need this empty `name`. You could go with `{}` simply, or even better `{ cssClass: 'w-48' }` to have some fixed width (it can be any other of course)

Reply:
I'd rather use your first suggestion of leaving it as `{}` as I think that having a fixed might use more space in the cell than what we actually need.